### ISSUE TITLE
Ajout d'un onglet "Etudes"

### ISSUE THREAD

**Issue by sylvain-m** [PnEcrins/GeoNature-atlas#96]
Les fiches espèces présentent actuellement 2 onglets : 
- communes (liste des communes avec lien vers les fiches communes)
- observateurs (liste des observateurs)

Dans le cadre de notre Atlas, il serait utile d'ajouter un 3e onglet permettant de lister les "études" et le nombre d'observations du Taxon pour chaque étude.

Cette notion d'étude (= Relevé dans SERENA) peut aussi être associée avec la notion de protocole de GeoNature.
Cela permettra, pour une espèce, de connaître la provenance des données. Comme notre base de données SERENA est inter-structure, et que le Relevé précise la structure, cela permettra de connaître également les structures sources de données.

Cette synthèse des études pourrait être utile également au niveau des "fiches communes".

**Comment by camillemonchicourt** [comment]
J'aurai tendance à dire que plus la notion d'étude (qui est assez technique et spécifique), c'est surtout la notion de structure qu'il faut bien faire apparaitre à partir du moment où GeoNature-atlas accueille des données de plusieurs structures. 
Pour notre part, dans un premier temps, nous avons développé l'outil pour y diffuser uniquement nos données. 
Néanmoins nous réfléchissons déjà à comment y intégrer des données de partenaires (dont plusieurs ont manifesté leur intérêt pour cela). 
C'est déjà possible d'intégrer des données de partenaires mais elles seront mélangées avec les autres, sans mention de source (https://github.com/PnEcrins/GeoNature-atlas/issues/7).

La première question que cela pose est de savoir si on les affiche sur les cartes au même niveau ou si elles affichables à la demande.
Autre question : doivent-elles être globalisées au niveau des stats ou faut-il dissocier les stats pour chaque structures ? Idem pour les graphiques. Idem quand on affiche les infos d'une maille. 

Selon moi, dissocier les stats pour chaque structure risque d'alourdir pas mal les interfaces. 
A minima, quand on clique sur une observation, en plus de l'observateur, il faudrait afficher sa structure et sur chaque fiche il faudrait un onglet STRUCTURES qui listerait les structures qui ont observé l'espèce et pour chacune le nombre d'observations de l'espece. Les autres stats restant globales pour rester simple et lisible.

**Comment by sylvain-m** [comment]
Effectivement, la notion de Structure est prioritaire : elle est même capitale pour notre Atlas.

Mais la notion d'étude ne me semble pas plus technique ou spécifique que l'Observateur.
Cela permettrait de rendre visible les (nombreuses depuis 1975) études du Parc, auxquelles sont rattachées chacune de nos observations.
D'ailleurs on pourrait même imaginer des fiches par études, listant l'ensemble des espèces de l'étude (mais là, c'est très très secondaire).

**Comment by gildeluermoz** [comment]
Comme le dit Camille, il faut être très vigilant quant à l'ajout de fonctionnalités nouvelles.
Un outil grand public doit resté simple et compréhensible facilement. Plus il y a de fonctionnalités, plus l'interface s'alourdit au détriment de l'ergonomie.
Il faut donc bien analyser le rôle de chacun des 2 outils : GeoNature et l'Atlas et le public ciblé par chacun d'eux.
Si un public spécialiste peut avoir des besoins poussés avec l'atlas, il faut ne faut pas que ces besoins se traduisent dans l'interface par une dégradation de la volontaire simplicité et de l'accessibilité au grand public.
S'il existe vraiment 2 besoins distincts, il faut peut-être carrément se poser la question d'un autre outil, intermédiaire entre GeoNature et l'atlas.

**Comment by sylvain-m** [comment]
Je suis tout à fait d'accord sur le fait qu'il faut que l'outil reste très ergonomique et "grand public".
Mais je persiste à penser que la notion d'études peut être grand public, et que l'ajout de cette fonctionnalité ne nuirait pas du tout à l'ergonomie globale : 
![geonature_etudes](https://cloud.githubusercontent.com/assets/11629210/21980471/c5febcd6-dbe3-11e6-8751-9a9eb7c895e7.png)
La logique des onglets sous la carte permet justement d'imaginer l'ajout d'un onglet, que l'utilisateur ouvre ou pas, sans que ça pollue l'interface inutilement.
Après, si une structure ne souhaite pas activer cet onglet, il suffirait de le mettre en paramètre.

Comme je sens bien que vous n'accrochez pas sur cette suggestion (c'est logique qu'on ait des attentes différentes ;) ), si je la fait développer, je ferai en sorte que ce soit activable ou non.

**Comment by camillemonchicourt** [comment]
Oui pourquoi pas. Sachant qu'il faut aussi un onglet STRUCTURES comme évoqué plus haut.

**Comment by sylvain-m** [comment]
Non, je pense que cet onglet structure pourrait justement, pour éviter la surcharge des onglets, se retrouver dans les études.
Voila ce que ça pourrait donner dans l'onglet études : 


- Parc naturel régional Normandie-Maine
    - Inventaire des Lichens de la Forêt Bidule : 10
    - Suivi du Site Natura 2000 "Machin chose" : 5
    - GeoNature PNRNM - Contact-Flore : 7
    - ...
- Association Faune et Flore de l'Orne
    - Données ponctelles AFFO : 5
    - GeoNature AFFO : 8
    - ...

Pour ce qui est de l'écriture du SQL de la (des) requête(s) correspondante(s), aucun souci de mon côté (et ça necessitera du nettoyage sur mesure dans notre Base, mais ça c'est pas un souci).
Par contre, au niveau de mes compétences, ça s'arrêtera là : mais comme je disais, si besoin on aura un peu de budget pour ça en 2017.

**Comment by camillemonchicourt** [comment]
Hum ces exemples me confortent dans l'idée que c'est vraiment pas une info grand public ! :-p

Mais au-delà de ça, je préfère vraiment un onglet STRUCTURES et éventuellement un autre pour les ETUDES. 
De plus avec ce fonctionnement groupé si 2 structures ont travaillé sur un même programme, ça doublonne le comptage...

Enfin, sur la source de cette info ETUDE, c'est plus dans la table `meta.bib_programmes` qu'il faudrait la prendre que dans `meta.t_protocoles`.

La notion de PROGRAMME est d'ailleurs en réflexion pour certainement l'ajouter dans le CONTACT FAUNE : https://github.com/PnEcrins/GeoNature/issues/138

**Comment by sylvain-m** [comment]
> La notion de PROGRAMME est d'ailleurs en réflexion pour certainement l'ajouter dans le CONTACT FAUNE : PnEcrins/GeoNature#138

Cela faisait parti effectivement des demandes que j'allais formuler très vite : cette notion de programme est très importante pour nous (et pas uniquement pour "Contact Faune", de manière générale pour toutes les observations/protocoles).

Elle rejoint bien notre notion de "Relevés" dans SERENA (nom de la Table dans SERENA, auquel nous nous sommes pliés, mais c'est bien le terme "Programme" qui est plus parlant).

> Hum ces exemples me confortent dans l'idée que c'est vraiment pas une info grand public ! :-p

Autant je suis d'accord qu'il faut que l'interface soit ergonomique dans la **forme**, autant dans le **contenu**, il est nécessaire d'accepter que par essence, une base de données d'Observations naturalistes contient des informations "techniques" pas très "grand public" : si on voulait vraiment que du grand public, pourquoi mettrait-on des noms latins ?
Dans le public cible de notre application, il y a certes le grand public, mais (chez nous en tout cas) il y a aussi les élus et les partenaires : pour eux, ces informations sont utiles, et bien disposées dans l'appli elles ne noient pas le grand public dans la complexité.
Et développer une autre application dédiée, plus technique, pour les partenaires et les élus ne me semble pas non plus judicieux, non ?

**Comment by camillemonchicourt** [comment]
> Et développer une autre application dédiée, plus technique, pour les partenaires et les élus ne me semble pas non plus judicieux, non ?

Oui c'e serait dommage mais ça dépend où on s'arrête... :-) et si on dissocie bien l'outil métier GeoNature (quitte à lui ajouter des choses) et l'outil de diffusion au grand public.

**Comment by sylvain-m** [comment]
C'est sûr que l'outil métier pourra certainement apporter d'autres réponses aux partenaires techniques. Mais pour ce qui concerne les élus, il est coutume de les considérer comme "grand public".
Et il nous semble qu'un maire pourrait être intéressé de connaître les différentes études "scientifiques" qui ont cours sur sa commune (ce qui est n'est pas plus technique que de savoir que _Carabus granulatus Linnaeus, 1758_ est présent).

(bon, je reconnais que j'insiste et paraît peut-être "têtu", mais entendez bien que c'est tout à fait cordial :) )

**Comment by camillemonchicourt** [comment]
Oui j'aime bien ces débats et c'est bien le but. :-)
Si ça n'intéresse que le maire alors ça me conforte que je n'en veux pas ! :-p
Mais un onglet ETUDES désactivable pour ceux qui veulent choyer leurs maires, pourquoi pas, si ça reste raisonnable :-D

**Comment by sylvain-m** [comment]
> Si ça n'intéresse que le maire alors ça me conforte que je n'en veux pas ! :-p

Pas que le maire : le maire, tous les élus du comité syndical, les partenaires (financiers) qui n'auront pas accès à GeoNature et même les collègues qui préfèrent l'ergonomie d'un outil "Grand public" qu'un outil métier comme GeoNature (même si l'ergonomie de GeoNature évoluera avec la V2/Leaflet...)

> Mais un onglet ETUDES désactivable pour ceux qui veulent choyer leurs maires, pourquoi pas, si ça reste raisonnable :-D

Cool 8-)

**Comment by camillemonchicourt** [comment]
OK donc cela confirme un problème de positionnement. Désolé d'insister pour ma part aussi, têtu aussi. :-D

C'est donc bien un besoin interne/partenaires qu'on va mettre dans un outil externe. 

Attention donc à rester raisonnable avec ce genre de détournements de l'outil.

**Comment by sylvain-m** [comment]
> C'est donc bien un besoin interne/partenaires qu'on va mettre dans un outil externe.

Pour nous, les partenaires sont dans la case "Externe", et pas dans la case "Interne". Ce n'est donc pas un détournement.
Et j'oubliais de mettre le grand public dans la liste des intéressés, mais comme je le disais, il nous semble autant utile de sensibiliser le grand public à la diversité des études scientifiques menées par le Parc, qu'à la présence de _Carabus granulatus Linnaeus, 1758_ sur sa commune. Ca fait partie de la vulgarisation aussi.

**Comment by camillemonchicourt** [comment]
OK
